mods/default/locale/default.eo.tr

```diff
@@ -167,14 +167,14 @@ Key=Ŝlosilo
 Furnace is empty=Forno estas malplena
 100% (output full)=100% (eligo estas plena)
 @1%=@1%
-Not cookable=Ne povas kuiri
+Not cookable=Nekuirebla
 Empty=Malplena
 Furnace active=Forno laboras
 Furnace inactive=Forno ne laboras
 (Item: @1; Fuel: @2)=(Objekto: @1; Brulaĵo: @2)
 Furnace=Forno
 Title:=Titolo
```

Comment:
What about the colons here? Are colons used differently in Esperanto?

Reply:
Are you talking about the missing one on the last line?
Because it is effectively a miss to be corrected.

Reply:
Shouldn't "Title:" be translated as "Titolo:" is what I'm trying to say (same for the line below that)